src/main/java/hudson/plugins/parameterizedtrigger/TriggerBuilder.java

```diff
@@ -134,15 +134,19 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher,
                         }
                         for (Future<Run> future : futures.get(p)) {
                             try {
-                                listener.getLogger().println("Waiting for the completion of " + HyperlinkNote.encodeTo('/'+ p.getUrl(), p.getFullDisplayName()));
-                                Run b = future.get();
-                                listener.getLogger().println(HyperlinkNote.encodeTo('/'+ b.getUrl(), b.getFullDisplayName()) + " completed. Result was "+b.getResult());
-                                BuildInfoExporterAction.addBuildInfoExporterAction(build, b.getParent().getFullName(), b.getNumber(), b.getResult());
-
-                                if(buildStepResult && config.getBlock().mapBuildStepResult(b.getResult())) {
-                                    build.setResult(config.getBlock().mapBuildResult(b.getResult()));
+                                if (future != null ) {
+                                    listener.getLogger().println("Waiting for the completion of " + HyperlinkNote.encodeTo('/'+ p.getUrl(), p.getFullDisplayName()));
+                                    Run b = future.get();
+                                    listener.getLogger().println(HyperlinkNote.encodeTo('/' + b.getUrl(), b.getFullDisplayName()) + " completed. Result was " + b.getResult());
+                                    BuildInfoExporterAction.addBuildInfoExporterAction(build, b.getParent().getFullName(), b.getNumber(), b.getResult());
+
+                                    if (buildStepResult && config.getBlock().mapBuildStepResult(b.getResult())) {
+                                        build.setResult(config.getBlock().mapBuildResult(b.getResult()));
+                                    } else {
+                                        buildStepResult = false;
+                                    }
                                 } else {
-                                    buildStepResult = false;
+                                    listener.getLogger().println("Skipping " + HyperlinkNote.encodeTo('/'+ p.getUrl(), p.getFullDisplayName()) + ". The project was disabled.");
```

Comment:
I can put ```The project was not triggered by some reason``` or 
```
The project was not triggered because one of possible cases happened: 
1) Project was disabled in moment then this build started 
2) Configuration of that project was not saved then this build started 
3) Project is not instance of "ParameterizedJobMixIn.ParameterizedJob"
4) Jenkins just does not like you (dispatcher extensions return null instead of running this project) :(
```
Or keep only first part `Skipping...`.